atomata/capstone_WebClient
# Selection page frontend

### Summary 
Enhancement of the selection page UI. 
Added Preview button that displays a gif of the selected apparatus.

### How to test?
1. Login with any username and password. 
2.  Create a new experience from the workbench page.
3. Click various preview buttons available and check if the gif is of the accurate apparatus.
4. Click one of the select buttons to select an apparatus and go to the editor page. 
5. Save an experience and click the return button on top to go back to the workbench page.
6. Repeat step 2 to 5 and finally logout.

### Checks
